source/UberCore/Networking/UberError.swift

```diff
@@ -1,5 +1,5 @@
 //
-//  RidesError.swift
```

Comment:
I think it makes sense to have the Error object in the base SDK, since all of our API's errors follow a standard format. I made the same recommendation to Ty in my Android SDK feedback. Any reason you think the Error object should live in Rides?

Reply:
I don't have a strong feeling on where it *should* belong, comment was just in the name of parity. There were dependency reasons Ty couldn't easily split it out, which is how we made the decision there. If we truly do have standard error formats it certainly makes sense to be able to reuse it. I'd just sync with Ty about it, there's a lot to be said for parity over purity.